Sources/Build/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ target_link_libraries(Build PUBLIC
   SPMBuildCore
   CYaml
   Yams
+  ArgumentParser
```

Comment:
This isn't strictly necessary yet, but I think it'll allow incremental adoption of ArgumentParser in the driver library